docs/custom-workload.md

```diff
@@ -0,0 +1,115 @@
+        securityContext:
+          runAsNonRoot: false
+          windowsOptions:
+            runAsUserName: "ContainerAdministrator"
+      nodeSelector:
+        kubernetes.io/os: windows
+```
+
+For more information about windows workloads, visit
+[the docs](<https://docs.openshift.com/container-platform/4.12/windows_containers/scheduling-windows-workloads.html>).
+
+## Configure the base image
+
+It is very important to set the correct base image version for your test workload.
+Each version of the WMCO uses a specific windows operating system version, and each workload has its own base image.
+
+To know which windows version your release uses, check the defaultSKU in
+[azure.go](<https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/providers/azure/azure.go#L21>)
+for the WMCO release you’re testing with.
+To know which base image your release uses, check the
+[getWindowsServerContainerImage()](<https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/network_test.go#L460>)
+function in network_test.go for the WMCO release you’re testing with.
+
+### Check the operating system on the cluster
+
+To be sure that your deployment’s base image corresponds with the cluster’s OS version, you should check what version
+of OS your cluster is running.
+
+### Edit your deployment
+
+Now that you know what operating system is running, you have to find the corresponding base image to set your
+deployment to. This is an incomplete table of which windows version corresponds with which powershell base image.
+
+| Windows Version                 | Base Image version                                   |
+|---------------------------------|------------------------------------------------------|
+| 2019-Datacenter-with-Containers | mcr.microsoft.com/powershell:lts-nanoserver-1809     |
+| 2022-datacenter-smalldisk       | mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022 |
+| datacenter-core-20h2-with-containers-smalldisk | mcr.microsoft.com/powershell:lts-nanoserver-20h2|
+
+If you are using the example deployment, a sed command can update the base image for you.
+
+```shell script
+export BASE_IMAGE=powershell:lts-nanoserver-ltsc2022
+sed -i 's/image:.*$/image: "mcr.microsoft.com\/'$BASE_IMAGE'"/' deployment.yaml
+```
+
+### Set the powershell executable
+
+Different base images have different executables for powershell. Since WMCO uses the powershell nanoserver images, the
+command should be set to pwsh.exe.
+Change the executable to pwsh.exe by running
+
+```shell script
+sed -i "s/powershell.exe/pwsh.exe/" deployment.yaml
+```
+
+## Deploy the winworker machine
+
+In order to generate and apply the windows MachineSet.yaml you must enter the top level of your wmco directory and run
+
+```shell script
+hack/machineset.sh apply
+```
+
+## Deploy the workload
+
+now you can deploy the workload by running
+
+```shell script
+oc apply -f deployment.yaml
+```
+
```

Comment:
Why is this here? The user should have deployed the Windows node to deploy a workload. Please remove.